src/components/Nav.js

```diff
@@ -1,20 +1,39 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { Bird } from 'phosphor-react'
+import { useNavigate } from 'react-router-dom'
 
 function Nav () {
+  const [nav, stickyNav] = useState(false)
+
+  function Sticky () {
+    if (window.scrollY >= 500) {
+      stickyNav(true)
+    } else {
+      stickyNav(false)
+    }
+  }
+  window.addEventListener('scroll', Sticky)
+
+  const navigate = useNavigate()
+  const routeChange = () => {
+    const path = '/chat'
```

Comment:
Why make a function for this. It's less code to just use the `navigate` function.
`navigate('/chat')`